### ISSUE TITLE
SyntaxError: Functions cannot be declared in a nested block in strict mode

### ISSUE THREAD

**Issue by FerrielMelarpis** [zhuowenli/vue-clipboards#32]
Hi @zhuowenli, thanks for the useful plugin.
I encountered an issue when testing components that use vue-clipboards.
PhantomJS emits this error:
```
PhantomJS 2.1.1 (Linux 0.0.0) ERROR
  {
    "message": "SyntaxError: Functions cannot be declared in a nested block in strict mode\nat webpack:///node_modules/vue-clipboards/dist/vue-clipboards.es.js:830:0 <- index.js:187467",
    "str": "SyntaxError: Functions cannot be declared in a nested block in strict mode\nat webpack:///node_modules/vue-clipboards/dist/vue-clipboards.es.js:830:0 <- index.js:187467"
  }
```
I think this is the cause of the issue:
https://github.com/zhuowenli/vue-clipboards/blob/1337d02d08090abbc88523d16f46e53c0b7e38cc/dist/vue-clipboards.es.js#L830-L874

It should have been wrapped with an IIFE. Not entirely sure what went wrong with the library's build files.

**Comment by FerrielMelarpis** [comment]
@zhuowenli Is this an issue on this repo or in https://github.com/egoist/bili? I'm not sure if this was the right place to file this bug.

**Comment by zhuowenli** [comment]
maybe I found the issue, but I can't really reproduce this. providing a mini repo to reproduce this can help me debug~

**Comment by FerrielMelarpis** [comment]
Hi @zhuowenli,
I created a repro here https://github.com/FerrielMelarpis/repro-vue-clipboards/pull/1
The test uses karma, mocha, sinon, chai and PhantomJS as browser

**Comment by zhuowenli** [comment]
fixed in `v1.2.3`
